packages/state-transition/src/block/slashValidator.ts

```diff
@@ -37,9 +37,12 @@ export function slashValidator(
   validator.withdrawableEpoch = Math.max(validator.withdrawableEpoch, epoch + EPOCHS_PER_SLASHINGS_VECTOR);
 
   const {effectiveBalance} = validator;
-  // TODO: could state.slashings be number?
+  // state.slashings is a number because it's reset per epoch in processSlashingsReset()
+  // for each epoch, there are 8704 max validators to slash so it's safe to use Number
+  // also we don't need to compute the total slashings from state.slashings, it's handled by totalSlashingsByIncrement in EpochCache
   const slashingIndex = epoch % EPOCHS_PER_SLASHINGS_VECTOR;
-  state.slashings.set(slashingIndex, state.slashings.get(slashingIndex) + BigInt(effectiveBalance));
+  state.slashings.set(slashingIndex, state.slashings.get(slashingIndex) + effectiveBalance);
```

Comment:
can `state.slashings.get(slashingIndex)` be undefined? may be worth adding `?? 0`

Reply:
I added it 👍